editor.planx.uk/src/@planx/components/MapAndLabel/Public.tsx

```diff
@@ -1,17 +1,62 @@
 import { alpha } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import { Feature } from "geojson";
 import { useStore } from "pages/FlowEditor/lib/store";
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { FONT_WEIGHT_SEMI_BOLD } from "theme";
 
 import Card from "../shared/Preview/Card";
 import CardHeader from "../shared/Preview/CardHeader";
-import { MapContainer } from "../shared/Preview/MapContainer";
+import { MapContainer, MapFooter } from "../shared/Preview/MapContainer";
 import { PublicProps } from "../ui";
 import { MapAndLabel } from "./model";
 
 type Props = PublicProps<MapAndLabel>;
 
+export type Boundary = Feature | undefined;
+
 function MapAndLabelComponent(props: Props) {
   const teamSettings = useStore.getState().teamSettings;
+  const [boundary, setBoundary] = useState<Boundary>();
```

Comment:
nit: for simplicity, I wouldn't export a type `Boundary` (unless there's a use case outside of this file??), I'd do this all in the same line like so: 
```
const [boundary, setBoundary] = useState<Feature | undefined>()
```